server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -68,13 +70,13 @@ test -d $RESULTS || doexit "Bad RESULTS=$RESULTS"
 # does not use any locking.
 
 # the link source and destination for this script
-linksrc=TODO
-linkdest=TO-COPY-SOS
+linksrc=TO-UNPACK
+linkdest=UNPACKED
 
 echo $TS
 
 # get the list of files we'll be operating on - sort them by size
-list=$TMP/pbench-unpack-tarballs.$$
+list=$UNPACK_PATH/pbench-unpack-tarballs.$$
 find -L $ARCHIVE/*/$linksrc -name '*.tar.xz' -printf "%s\t%p\n" 2>/dev/null | grep -v DUPLICATE | sort -n > $list
```

Comment:
Testing showed that this is wrong: the -L leads to too many levels of symlinks to be resolved. It should be replaced by this:

     list=$UNPACK_PATH/pbench-unpack-tarballs.$$
     find $ARCHIVE/*/$linksrc -name '*.tar.xz' -printf "%s\t%p\n" 2>/dev/null | grep -v DUPLICATE |
         sed 's/\/'$linksrc'//' | sort -n > $list